src/messages.rs

```diff
@@ -267,6 +263,9 @@ impl RoutingMessage {
 #[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Debug, RustcEncodable, RustcDecodable)]
 pub struct SignedMessage {
     encoded_body : Vec<u8>,
+    claimant     : Address,
```

Comment:
Just realized that this field will be redundant for cases when `from_authority` is `ManagedNode` and `undefined` when it is `Client`.